src/constraint.rs

```diff
@@ -171,6 +173,23 @@ impl Constraint {
         Constraint::Fields(FieldsConstraint::new(fields.collect(), true))
     }
 
+    /// Creates a [Constraint::ValidValues] using the [OwnedElements] specified inside it
```

Comment:
I think `[OwnedElements]` will produce a broken doc link.

```suggestion
    /// Creates a [Constraint::ValidValues] using the [OwnedElement]s specified inside it
```